frontend/src/assets/css/custom_components_theme.scss

```diff
@@ -13,13 +13,860 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 @import "variables";
 @import "fonts";
 @import "scrollbars";
 
-// Basics.
+/* Boostrap styles */
+
+*,
```

Comment:
Can you add a comment explaining why we're copying this out of bootstrap?

Reply:
Done!